### ISSUE TITLE
about learning rate

### ISSUE THREAD

**Issue by songqi-github** [CoinCheung/BiSeNet#98]
how do you set the value of warmup_steps and warmup_start_lr? Could you please share any experience about that？ THX！

**Comment by CoinCheung** [comment]
From my observation, the results is not too sensitive tp different warmup strategies. Warmup is mentioned in this [paper](https://arxiv.org/abs/1706.02677) to stablize training when learning rate is large. If your batchsize is not too small and you do not have a problem with training stability during the first few epoches, you have rich freedom to determine how the learning rate is warmed-up.

**Comment by CoinCheung** [comment]
Since I have explained my understanding of warmup, and you have no further puzzlement on this, I am closing this.
